state/subnets.go

```diff
@@ -149,10 +149,11 @@ func (s *Subnet) AvailabilityZones() []string {
 	return s.doc.AvailabilityZones
 }
 
-// SpaceName returns the space the subnet is associated with. If the subnet is
-// not associated with a space it will be the empty string.
+// SpaceName returns the space the subnet is associated with.  If no
+// space is associated, return the default space and log an error.
 func (s *Subnet) SpaceName() string {
```

Comment:
Aside from tests, it is only used in status, and we can just look it up by ID there and remove the entity cross-dependency here.

Reply:
added a todo